whist/server/api/game/action.py

```diff
@@ -62,6 +62,7 @@ def start_game(game_id: str, model: StartModel,
             headers={"WWW-Authenticate": "Basic"},
         ) from ready_error
     else:
+        logger.info(user.username + " has started " + game)
```

Comment:
The reason for failing is, that logger is not defined. You still need to import the logging module and get the logger but configure the level for example.

Reply:
@Segelzwerg  so i would just need to import logging, use getLogger(), and add level? Would I need a formatter?

For example, out of these lines, which ones would I need.

import logging
import sys

logger = logging.getLogger(__name__)
stream_handler = logging.StreamHandler(sys.stdout)
formatter = logging.Formatter("%(levelname)s:%(message)s")
stream_handler.setFormatter(formatter)
logger.addHandler(stream_handler)
logger.setLevel(logging.INFO)